assets/sass/globals/forms.scss

```diff
@@ -149,7 +149,19 @@ $errorColor: #b00;
     padding: 0.5em;
 }
 
-.ff-radio,
+.ff-radio {
```

Comment:
Mirrors checkbox styles, there are no radio options on the RC form so I think these were just never styled.